<!DOCTYPE html>
<html lang="en">
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-135435972-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-135435972-1');
</script>

<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Goodarzi Lab @ UCSF</title>
<meta name="description" content="Goodarzi Lab @ UCSF">

<!-- core CSS -->
<link href="css/bootstrap.min.css" rel="stylesheet">
<link href="css/bootstrap-social.css" rel="stylesheet">
<link rel="stylesheet" href="css/academicons.min.css"/> 
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
<link href="css/prettyPhoto.css" rel="stylesheet">
<link href="css/main.css" rel="stylesheet">
<link href="css/responsive.css" rel="stylesheet">
<link rel="shortcut icon" type="image/x-icon" href="favicon-flask.ico" />
<!--[if lt IE 9]>
    <script src="js/html5shiv.js"></script>
    <script src="js/respond.min.js"></script>
    <![endif]-->
</head>
<!--/head-->
<body>
<header id="header">
  <nav class="navbar navbar-inverse" role="banner">
    <div class="container">
      <div class="navbar-header">
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
        <a class="navbar-brand" href="index.html"><i class="fa fa-flask"></i> Goodarzi Lab</a> 
      </div>
      <div class="collapse navbar-collapse navbar-right">
        <ul class="nav navbar-nav">
          <li><a href="index.html">Home</a></li>
          <li><a href="research.html">Research</a></li>
          <li class="active"><a href="publications.html">Publications</a></li>
          <li><a href="people.html">People</a></li>
          <li><a href="software.html">Software</a></li>
          <li><a href="donate.html">Donate</a></li>
          <li><a href="positions.html">Positions</a></li>
          <li><a href="contact-us.html">Contact</a></li>
        </ul>
      </div>
    </div>
    <!--/.container--> 
  </nav>
  <!--/nav--> 
</header>
<!--/header-->
<div class="color-border"> </div>
<section id="Publications">
  <div class="container" style='font-family: Verdana;'>

    <div class="center">
      <h1>Publications</h1>
    </div>
    <!--/.center-->
    <div class="center">
    <h2>Preprints</h2>
    </div>
    <p><li>Arshadi AK*, Salem M*, Karner H*, Garcia K, Arab A, Yuan JS, <strong>Goodarzi H</strong> (2023). Functional microRNA-Targeting Drug Discovery by Graph-Based Deep Learning. *Contributed equally; <a href="https://www.biorxiv.org/content/10.1101/2023.01.13.524005v1"><i class="ai ai-biorxiv"></i></a></li></p>

    <p><li>Ge A*, Arab A*, Navickas A, Fish L, Garcia K, Asgharian H, <strong>Goodarzi H</strong>&#x271D, Gilbert L&#x271D (2022). A multiomics approach reveals RNA dynamics promote cellular sensitivity to DNA hypomethylation. *Contributed equally; &#x271D Co-corresponding authors <a href="https://www.biorxiv.org/content/10.1101/2022.12.14.518457v1"><i class="ai ai-biorxiv"></i></a></li></p>

    <p><li>Nanda S*, Wu K*, Kasinathan S*&#x271D, Ostrowski MS, Clugston AS, Satpathy AS, Sweet-Cordero AE, <strong>Goodarzi H</strong>, Ramani V&#x271D (2022). Sensitive multimodal profiling of native DNA by transposase-mediated single-molecule sequencing. *Contributed equally; &#x271D Co-corresponding authors <a href="https://www.biorxiv.org/content/10.1101/2022.08.07.502893v2"><i class="ai ai-biorxiv"></i></a></li></p>

    <p><li>Raths F*, Karimzadeh M*, Ing N, Martinez A, Qu Y, Lee T, Mulligan B, Devkota S, Wang B, Giuliano AE, Bose S, <strong>Goodarzi H</strong>&#x271D, Ray EC&#x271D, Cui X&#x271D, Knott SRV&#x271D (2022). The molecular consequences of androgen activity in the human breast. *Contributed equally; &#x271D Co-corresponding authors <a href="https://www.biorxiv.org/content/10.1101/2022.04.22.489095v1"><i class="ai ai-biorxiv"></i></a></li></p>

    <p><li>Culbertson B*, Garcia K*, Markett D, Asgharian H, Chen L, Fish L, Navickas A, Yu J, Woo B, Nanda S, Rabinowitz J, and <strong>Goodarzi H</strong> (2021). A sense-antisense RNA interaction promotes breast cancer metastasis via regulation of NQO1 expression. *Contributed equally <a href="https://www.biorxiv.org/content/10.1101/2021.10.08.463652v1"><i class="ai ai-biorxiv"></i></a></li></p>

    <p><li>Navickas A*, Asgharian H*, Winkler J, Fish L, Garcia K, Markett D, Dodel M, Culbertson B, Miglani S, Joshi T, Nguyen P, Zhang S, Stevers N, Hwang H, Mardakheh F, Goga A, and <strong>Goodarzi H</strong> (2021). An mRNA processing pathway suppresses metastasis by governing translational control from the nucleus. *Contributed equally <a href="https://www.biorxiv.org/content/10.1101/2021.10.04.463118v1"><i class="ai ai-biorxiv"></i></a></li></p>

    </br></br>

    <div class="center">
    <h2>2022</h2>
    </div>
    <p><li>Earnest-Noble LB, Hsu D, Chen S, Asgharian H, Nandan M, Passarelli MC, <strong>Goodarzi H</strong>&#x271D, Tavazoie SF&#x271D (2022). Two isoleucyl tRNAs that decode synonymous codons divergently regulate breast cancer metastatic growth by controlling translation of proliferation-regulating genes. <strong><em>Nature Cancer</em></strong> https://doi.org/10.1038/s43018-022-00469-9. &#x271D Co-corresponding authors <a href="https://www.nature.com/articles/s43018-022-00469-9"><i class="fa fa-chrome"></i></a><a href="https://www.biorxiv.org/content/10.1101/2021.04.22.440519v1.full"><i class="ai ai-biorxiv"></i></a></li></p>
    <p><li>Passarelli MC, Pinzaru AM, Asgharian H, Liberti MV, Heissel S, Molina H, <strong>Goodarzi H</strong>&#x271D, Tavazoie SF&#x271D (2022). Leucyl-tRNA synthetase is a tumour suppressor in breast cancer and regulates codon-dependent translation dynamics. <strong><em>Nature Cell Biol</em></strong> 13(1):167. &#x271D Co-corresponding authors <a href="https://www.nature.com/articles/s41556-022-00856-5"><i class="fa fa-chrome"></i></a></li></p>
    </br></br>
    <div class="center">
    <h2>2021</h2>
    </div>
    <p><li>Yogodzinski C, Arab A, Pritchard JR, <strong>Goodarzi H</strong>, Gilbert LA (2021). A global cancer data integrator reveals principles of synthetic lethality, sex disparity and immunotherapy. <strong><em>Genome Med</em></strong> 13(1):167 <a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-021-00987-8">
      <i class="fa fa-chrome"></i></a></li></p>

    <p><li>Xu Y, Huangyang P, Wang Y, Xue L, Devericks E, Nguyen HG, Yu X, Oses-Prieto JA, Burlingame AL, Miglani S, <strong>Goodarzi H</strong>, and Ruggero D (2021). ERα is an RNA-binding protein sustaining tumor cell survival and drug resistance. <strong><em>Cell</em></strong> 184(20):5215-5229 <a href="https://www.sciencedirect.com/science/article/pii/S0092867421010473"><i class="fa fa-chrome"> </i></a></li></p>

    <p><li>Chen WC, To MD, Westcott PMK, Delrosario R, Kim IJ, Philips M, Tran Q, Bollam SR, <strong>Goodarzi H</strong>, Bayani N, Mirzoeva O, Balmain A (2021). Targeting KRAS4A splicing through the RBM39/DCAF15 pathway inhibits cancer stem cells. <strong><em>Nat Communications</em></strong> 12(1):4288 <a href="https://www.nature.com/articles/s41467-021-24498-7"><i class="fa fa-chrome"> </i></a></li></p>

    <p><li>Kuzuoglu-Ozturk D, Hu Z, Rama M, Devericks E, Weiss J, Chiang GG, Worland ST, Brenner SE, <strong>Goodarzi H</strong>, Gilbert LA, Ruggero D. (2021). Revealing molecular pathways for cancer cell fitness through a genetic screen of the cancer translatome. <strong><em>Cell Rep</em></strong> 35(13):109321 <a href="https://www.sciencedirect.com/science/article/pii/S2211124721006975"><i class="fa fa-chrome"> </i></a></li></p>

    <p><li>Fish L, Khoroshkin M, Navickas A, Garcia K, Culbertson B, Hänisch B, Zhang S, Nguyen HCB, Soto L, Dermit M, Mardakheh FK, Molina H, Alarcón C, Najafabadi HS, and <strong>Goodarzi H</strong> (2021). A pro-metastatic splicing program regulated by SNRPA1 interactions with structured RNA elements. <strong><em>Science</em></strong> 372 (6543) eabc7531 <a href="https://science.sciencemag.org/content/372/6543/eabc7531"><i class="fa fa-chrome"></i></a></li></p>

     <p><li>Huh D, Passarelli MC, Gao J, Dusmatova SN, Goin C, Fish L, Pinzaru AM, Molina H, Ren Z, McMillan EA, Asgharian H, <strong>Goodarzi H</strong>, Tavazoie SF (2021). A stress-induced tyrosine-tRNA depletion response mediates codon-based translational repression and growth suppression.<strong><em>The EMBO Journal</em></strong> 40(2):e106696 <a href="https://www.embopress.org/doi/full/10.15252/embj.2020106696"><i class="fa fa-chrome"></i></a></li></p>

     <p><li>Zhu XG, Chudnovskiy A, Baudrier L, Prizer B, Liu Y, Ostendorf BN, Yamaguchi N, Arab A, Tavora B, Timson R, Heissel S, de Stanchina E, Molina H, Victora GD,      <strong>Goodarzi H</strong>, Birsoy K (2021). Functional Genomics In Vivo Reveal Metabolic Dependencies of Pancreatic Cancer Cells. <strong><em>Cell Metabolism</em></strong> 33(1):211-221.e6 <a href="https://www.sciencedirect.com/science/article/pii/S1550413120305507"><i class="fa fa-chrome"></i></a></li></p>


     </br></br>
  
    <div class="center">
    <h2>2016 - 2020</h2>
    </div>

    <p><li>Samuel RM, Majd H, Richter M, et al, Ott M, <strong>Goodarzi H</strong>&#x271D, Fattahi F&#x271D (2020). Androgen Signaling Regulates SARS-CoV-2 Receptor Levels and Is Associated with Severe COVID-19 Symptoms in Men. <strong><em>Cell Stem Cell</em></strong> 27:876-889. &#x271D Co-corresponding authors <a href="https://www.cell.com/cell-stem-cell/fulltext/S1934-5909(20)30547-6"><i class="fa fa-chrome"></i></a></li></p>

    <p><li>Zhu, X.G., Chudnovskiy, A., Baudrier, L. et al, <strong>Goodarzi H</strong>, and Birsoy K (2020). Functional Genomics In Vivo Reveal Metabolic Dependencies of Pancreatic Cancer Cells. <strong><em>Cell Metab</em></strong> doi.org/10.1016/j.cmet.2020.10.017 <a href="https://www.sciencedirect.com/science/article/pii/S1550413120305507"><i class="fa fa-chrome"></i></a></li></p>
    <p><li>Tavora, B., Mederer, T., Wessel, K.J. et al, <strong>Goodarzi H</strong>, and Tavazoie SF (2020). Tumoural activation of TLR3–SLIT2 axis in endothelium drives metastasis. <strong><em>Nature</em></strong> 586:299-304. <a href="https://www.nature.com/articles/s41586-020-2774-y"><i class="fa fa-chrome"></i></a></li></p>
    <p><li>Yu J, Naviskas A, Asgharian H, Culbertson B, Fish L, Garcia K, Olegario JP, Dermit M, Dodel M, Hanisch B, Liu Y, Weinberg EM, Dienstmann R, Warren RS, Mardakheh F, and <strong>Goodarzi H</strong> (2020). RBMS1 suppresses colon cancer metastasis through targeted stabilization of its mRNA regulon. <strong><em>Cancer Disc</em></strong> 10:1410–23 <a href="https://www.biorxiv.org/content/10.1101/2020.01.22.916205v1"><i class="ai ai-biorxiv"></i></a>  <a href="https://cancerdiscovery.aacrjournals.org/content/early/2020/06/06/2159-8290.CD-19-1375"><i class="fa fa-chrome"></i></a></li></p>
    <p><li>Fish L, Navickas A, Culbertson B, Xu Y, Nguyen HCB, Zhang S, Hochman M, Okimoto R, Dill BD, Molina H, Najafabadi HS, Alarcon C, Ruggero D, and <strong>Goodarzi H</strong> (2019). Nuclear TARBP2 Drives Oncogenic Dysregulation of RNA Splicing and Decay. <strong><em>Mol Cell</em></strong> 75(5), 967-81. <a href="https://doi.org/10.1101/389213"><i class="ai ai-biorxiv"></i></a>  <a href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(19)30434-4"><i class="fa fa-chrome"></i></a></li></p>
    <p><li>Litterman AJ, Kageyama R, Le Tonqueze O, Zhao W, Gagnon JD, <strong>Goodarzi H</strong>, Erle D, and Ansel M (2019). A massively parallel 3′ UTR reporter assay reveals relationships between nucleotide content, sequence conservation, and mRNA destabilization. <strong><em>Genome Res</em></strong> doi:10.1101/gr.242552.118. <a href="https://genome.cshlp.org/content/early/2019/05/31/gr.242552.118"><i class="fa fa-chrome"></i></a></li></p>
    <p><li>Mauer J, Sindelar M, Despic V, Guez T, Hawley BR, Vasseur J, Rentmeister A, Gross S, Pellizzoni L, Debart F, <strong>Goodarzi H</strong>, Jaffrey S (2019). FTO controls reversible m6Am RNA methylation during snRNA biogenesis. <strong><em>Nature Chem Biol</em></strong> 15, 340–347. <a href="https://www.nature.com/articles/s41589-019-0231-8"><i class="fa fa-chrome"></i></a></li></p>
    <p><li>Fish L, Zhang S, Yu J, Culbertson B, Zhou A, Goga A, <strong>Goodarzi H</strong> (2018). Cancer cells exploit an orphan RNA to drive metastatic progression. <strong><em>Nature Med</em></strong> 24, 1743-51. <a href="https://www.nature.com/articles/s41591-018-0230-4"><i class="fa fa-chrome"></i></a></li></p>
    <p><li>Knott SRV, Wagenblast E, Khan S, et al., <strong>Goodarzi H</strong>, Poulogiannis G, Hannon G (2018). Asparagine bioavailability governs metastasis in a model of breast cancer. <strong><em>Nature</em></strong> 554:378–381. <a href="https://www.nature.com/articles/nature25465"><i class="fa fa-chrome"></i></a></li></p>
    <p><li>Alkallas R, Fish L, <strong>Goodarzi H</strong>, Najafabadi HS (2017). Inference of RNA decay rate from transcriptional profiling highlights the regulatory programs of Alzheimer’s disease. <strong><em>Nature Comm</em></strong>  8: 909. <a href="https://www.nature.com/articles/s41467-017-00867-z"><i class="fa fa-chrome"></i></a></li></p>
    <p><li><strong>Goodarzi H</strong> (2017). Charting the “unknown unknowns” of cancer progression. <strong><em>Sci Transl Med</em></strong>  9 (400), eaao0959. <a href="http://stm.sciencemag.org/content/9/400/eaao0959.full?ijkey=l3dBm/4zTI6A6&keytype=ref&siteid=scitransmed"><i class="fa fa-chrome"></i></a></li></p>
    <p><li>Kingham PT, Nguyen CB, Zheng J, Konstantinidis IT, Sadot E, Shia J, Kuk D, Zhang S, Saltz L, D’Angelica M, Jarnagin WR, <strong>Goodarzi H</strong>&#x271D, Tavazoie SF&#x271D (2016). MicroRNA-203 predicts human survival after resection of colorectal liver metastasis. <strong><em>Oncotarget</em></strong> 8:18821-31. &#x271D Co-corresponding authors <a href="http://www.impactjournals.com/oncotarget/misc/linkedout.php?pii=13816"><i class="fa fa-chrome"></i></a></li></p>
    <p><li><strong>Goodarzi H</strong>*&#x271D, Nguyen HCB, Zhang S, Dill BD, Molina H, Tavazoie SF&#x271D (2016). Modulated expression of specific tRNAs drives gene expression and cancer progression. <strong><em>Cell</em></strong> 165, 1416-1427. *Contributed equally; &#x271D Co-corresponding authors <a href="http://www.cell.com/cell/fulltext/S0092-8674(16)30649-3"><i class="fa fa-chrome"></i></a></li></p>
    <p><li>Lee H, <strong>Goodarzi H</strong>, Tavazoie SF, Alarcon CR (2016). TMEM2 Is a SOX4-Regulated Gene That Mediates Metastatic Migration and Invasion in Breast Cancer. <strong><em>Cancer Res</em></strong> 76(17):4994-5005.<a href="http://cancerres.aacrjournals.org/cgi/pmidlookup?view=long&pmid=27328729"><i class="fa fa-chrome"></i></a></li></p>
    <p><li>Hwang HW, Park CY, <strong>Goodarzi H</strong>, Fak JJ, Mele A, Moore MJ, Saito Y, Darnell RB (2016). PAPERCLIP Identifies MicroRNA Targets and a Role of CstF64/64tau in Promoting Non-canonical poly(A) Site Usage. <strong><em>Cell Rep</em></strong> 15(2):423-435.<a href="http://www.sciencedirect.com/science/article/pii/S2211124716302649"><i class="fa fa-chrome"></i></a></li></p>
    <p><li>Nguyen A, Yoshida M, <strong>Goodarzi H</strong>, Tavazoie SF (2016). Highly variable cancer subpopulations that exhibit enhanced transcriptome variability and metastatic fitness. <strong><em>Nature Comm</em></strong> 7: 11246.<a href="http://www.nature.com/ncomms/2016/160503/ncomms11246/full/ncomms11246.html"><i class="fa fa-chrome"></i></a></li></p>
    <p><li>Fish L, Pencheva N*, <strong>Goodarzi H</strong>*, Tran H, Yoshida M, Tavazoie SF (2016). Muscleblind-like 1 suppresses breast cancer metastatic colonization and stabilizes metastasis suppressor transcripts. <strong><em>Genes &amp; Dev</em></strong> 30: 386-398. *Contributed equally <a href="http://genesdev.cshlp.org/content/30/4/386.short"><i class="fa fa-chrome"></i></a></li></p>


    </br></br>
    <div class="center">
    <h2>2012 - 2015</h2>
    </div>

    <p><li>Alarcon C, <strong>Goodarzi H</strong>, Lee H, Liu X, Tavazoie S, Tavazoie SF (2015). HNRNPA2B1 Is a Mediator of m6A-Dependent Nuclear RNA Processing Events. <strong><em>Cell</em></strong> 162: 1299-1308. <a href="http://www.cell.com/cell/abstract/S0092-8674(15)01024-7"><i class="fa fa-chrome"></i></a></li></p>
    
    <p><li><strong>Goodarzi H</strong>, Liu X, Nguyen HCB, Fish L, Tavazoie SF (2015). Endogenous tRNA-Derived Fragments Suppress Breast Cancer Progression via YBX1 Displacement. <strong><em>Cell</em></strong> 161: 790-802. <a href="http://www.cell.com/cell/abstract/S0092-8674(15)00318-9"><i class="fa fa-chrome"></i></a></li></p>

    <p><li>Furlow PW, Zhang S, Soong ST, Halberg N, <strong>Goodarzi H</strong>, Mangrum C, Wu G, Elemento O, Tavazoie S (2015). Mechanosensitive pannexin-1 channels mediate microvascular metastatic cell survival. <strong><em>Nature Cell Bio</em></strong> 17, 943-952. <a href="http://www.nature.com/ncb/journal/v17/n7/abs/ncb3194.html"><i class="fa fa-chrome"></i></a></li></p>

    <p><li>Alarcon CR, Lee H*, <strong>Goodarzi H</strong>*, Halberg N, Tavazoie S (2015). N<sup>6</sup>-methyladenosine marks primary microRNAs for processing. <strong><em>Nature</em></strong> 519, 482-485. *Contributed equally <a href="http://www.nature.com/nature/journal/v519/n7544/full/nature14281.html?WT.ec_id=NATURE-20150326"><i class="fa fa-chrome"></i></a></li></p>

    <p><li><strong>Goodarzi H</strong>, Zhang S, Buss CG, Fish L, Tavazoie S, Tavazoie SF (2014). Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins. <strong><em>Nature</em></strong> 513, 255-260 <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature13466.html"><i class="fa fa-chrome"></i></a></li></p>

    <p><li>Oikonomou P*, <strong>Goodarzi H</strong>*, Tavazoie S (2014). Systematic Identification of Regulatory Elements in Conserved 3? UTRs of Human Transcripts. <strong><em>Cell Reports</em></strong> 7(1): 281-292. *Contributed equally <a href="http://www.cell.com/cell-reports/fulltext/S2211-1247(14)00159-4"><i class="fa fa-chrome"></i></a></li></p>

    <p><li>Freddolino PL*, <strong>Goodarzi H</strong>*, Tavazoie S (2013). Revealing the genetic basis of natural bacterial phenotypic divergence. <strong><em>J. Bacteriol.</em></strong> doi:10.1128/JB.01039-13 *Contributed equally <a href="http://jb.asm.org/content/early/2013/12/04/JB.01039-13"><i class="fa fa-chrome"></i></a></li></p>

    <p><li>Chiu IM, Morimoto ETA, <strong>Goodarzi H</strong>, et al., Tavazoie S, Myers RM, Maniatis T (2013). A Neurodegeneration-Specific Gene-Expression Signature of Acutely Isolated Microglia from an Amyotrophic Lateral Sclerosis Mouse Model. <strong><em>Cell Reports</em></strong> 4(2): 385-401 <a href="http://www.cell.com/cell-reports/fulltext/S2211-1247(13)00296-9"><i class="fa fa-chrome"></i></a></li></p>

    <p><li>Freddolino PL*, <strong>Goodarzi H</strong>*, Tavazoie S (2012). Fitness landscape transformation through a single amino acid change in the Rho terminator. <strong><em>PLoS Genetics</em></strong> Vol. 8, e1002744. *Contributed equally <a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1002744"><i class="fa fa-chrome"></i></a></li></p>

    <p><li><strong>Goodarzi H</strong>, Najafabadi HS, Oikonomou P, Greco TM, Fish L, Salavati R, Cristea IM, Tavazoie S (2012). Systematic discovery of structural elements governing stability of mammalian messenger RNAs. <strong><em>Nature</em></strong> 485, 264-268. <a href="http://www.nature.com/nature/journal/v485/n7397/full/nature11013.html"><i class="fa fa-chrome"></i></a></li></p>

    </br></br>

    <div class="center">
    <h2>2007 - 2011</h2>
    </div>  

    <p><li> Korpal M, Ell BJ, Buffa FM, Ibrahim T, Blanco MA, Celia-Terrassa T, Mercatali L, Khan Z, <strong>Goodarzi H</strong>, et al. (2011). Direct targeting of Sec23a by miR-200s influences cancer cell secretome and promotes metastatic colonization. <strong><em>Nature Med</em></strong> 17(9):1048-9. <a href="http://www.nature.com/nm/journal/vaop/ncurrent/full/nm.2401.html"><i class="fa fa-chrome"></i></a></li></p>

    <p><li><strong>Goodarzi H</strong>, Bennet BD, et al. (2010). Regulatory and metabolic rewiring during laboratory evolution of ethanol tolerance in <em>E. coli</em>. <strong><em>Mol Syst Biol</em></strong> 6:278. <a href="http://www.nature.com/msb/journal/v6/n1/full/msb201033.html"><i class="fa fa-chrome"></i></a></li></p>

    <p><li><strong>Goodarzi H</strong>*, Elemento O*, Tavazoie S (2009). Revealing Global Regulatory Perturbations across Human Cancers. <strong><em>Mol Cell</em></strong> 36:900-11. *contributed equally <a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(09)00857-0"><i class="fa fa-chrome"></i></a></li></p>

    <p><li>Najafabadi HS*, <strong>Goodarzi H</strong>*, Salavati R (2009).  Universal function-specificity of codon usage. <strong><em>Nucl Acids Res</em></strong> 37(21):7014-7023. *contributed equally <a href="http://nar.oxfordjournals.org/cgi/content/full/gkp792"><i class="fa fa-chrome"></i></a></li></p>

    <p><li><strong>Goodarzi H</strong>, Hottes AK, Tavazoie S (2009). Global discovery of adaptive mutations. <strong><em>Nat Methods</em></strong> 6(8):581-3. <a href="http://www.nature.com/nmeth/journal/v6/n8/abs/nmeth.1352.html"><i class="fa fa-chrome"></i></a></li></p>

    <p><li>Amini S, <strong>Goodarzi H</strong>, Tavazoie S (2009). Genetic dissection of an exogenously induced biofilm in laboratory and clinical isolates of <em>E. coli</em>. <strong><em>PLoS Pathogens</em></strong> 5:e1000432. <a href="http://www.plospathogens.org/article/info%3Adoi%2F10.1371%2Fjournal.ppat.1000432"><i class="fa fa-chrome"></i></a></li></p>

    <p><li><strong>Goodarzi H</strong>, Torabi N, Najafabadi HS, Archetti M (2008). Amino acid and codon usage profiles: adaptive changes in the frequency of amino acids and codons. <strong><em>Gene</em></strong> 407(1-2):30-41. <a href="https://www.sciencedirect.com/science/article/pii/S0378111907004982"><i class="fa fa-chrome"></i></a></li></p>

    <p><li>Elahi E, Shafaghati Y, Asadi S, Absalan F, <strong>Goodarzi H</strong>, et al. (2007). Intragenic SNP haplotypes associated with 84dup18 mutation in TNFRSF11A in four FEO pedigrees suggest three independent origins for this mutation. <strong><em>J Bone Miner Metab</em></strong> 25(3):159-64. <a href="http://www.springerlink.com/content/68w0g65743860562/"><i class="fa fa-chrome"></i></a></li></p>

    <p><li><strong>Goodarzi H</strong>, Katanforoush A, Torabi N, Najafabadi HS (2007). Solvent accessibility, residue charge and residue volume, the three ingredients of a robust amino acid substitution matrix. <strong><em>J Theor Biol</em></strong> 245(4):715-25. <a href="https://www.sciencedirect.com/science/article/pii/S002251930600573X"><i class="fa fa-chrome"></i></a></li></p>

    <p><li>Torabi N, <strong>Goodarzi H</strong>, Shateri Najafabadi H. (2007). The case for an error minimizing set of coding amino acids. <strong><em>J Theor Biol</em></strong> 244(4):737-44. <a href="https://www.sciencedirect.com/science/article/pii/S0022519306004310"><i class="fa fa-chrome"></i></a></li></p>

    </br></br>
    <div class="center">
    <h2>2004 - 2006</h2>
    </div>

    <p><li>Najafabadi HS, <strong>Goodarzi H</strong>, Torabi N, Banihosseini SS.(2006). Applying a neural network to predict the thermodynamic parameters for an expanded nearest-neighbor model. <strong><em>J Theor Biol</em></strong> 238(3):657-65. <a href="https://www.sciencedirect.com/science/article/pii/S0022519305002729"><i class="fa fa-chrome"></i></a></li></p>

    <p><li>Marashi SA, <strong>Goodarzi H</strong>, Sadeghi M, Eslahchi C, Pezeshk H (2006). Importance of RNA secondary structure information for yeast donor and acceptor splice site predictions by neural networks. <strong><em>Comput Biol Chem</em></strong> 30(1):50-7. <a href="https://www.sciencedirect.com/science/article/pii/S1476927105001106"><i class="fa fa-chrome"></i></a></li></p>

    <p><li><strong>Goodarzi H</strong>, Shateri Najafabadi H, Torabi N (2005). On the coevolution of genes and genetic code. <strong><em>Gene</em></strong> 362:133-40. <a href="https://www.sciencedirect.com/science/article/pii/S0378111905005020"><i class="fa fa-chrome"></i></a></li></p>

    <p><li>Najafabadi HS, <strong>Goodarzi H</strong>, Torabi N (2005). Optimality of codon usage in Escherichia coli due to load minimization. <strong><em>J Theor Biol</em></strong> 237(2):203-9. <a href="https://www.sciencedirect.com/science/article/pii/S0022519305001621"><i class="fa fa-chrome"></i></a></li></p>

    <p><li><strong>Goodarzi H</strong>, Najafabadi HS, Hassani K, Nejad HA, Torabi N (2005). On the optimality of the genetic code, with the consideration of coevolution theory by comparison of prominent cost measure matrices. <strong><em>J Theor Biol</em></strong> 235(3):318-25. <a href="https://www.sciencedirect.com/science/article/pii/S0022519305000378"><i class="fa fa-chrome"></i></a></li></p>

    <p><li><strong>Goodarzi H</strong>, Shateri Najafabadi H, Torabi N (2005). Designing a neural network for the constraint optimization of the fitness functions devised based on the load minimization of the genetic code. <strong><em>Biosystems</em></strong> 81(2):91-100. <a href="https://www.sciencedirect.com/science/article/pii/S0303264705000080"><i class="fa fa-chrome"></i></a></li></p>

    <p><li><strong>Goodarzi H</strong>, Nejad HA, Torabi N (2005). On the optimality of the genetic code, with the consideration of termination codons. <strong><em>Biosystems</em></strong> 77(1-3):163-73. <a href="https://www.sciencedirect.com/science/article/pii/S0303264704000942"><i class="fa fa-chrome"></i></a></li></p>
  </div>
  <!--/.container--> 
</section>
<!--/#pulications--> 
    
<section id="get-started">
  <div class="container">
    <div class="row">
      <div class="col-sm-12">
        <h2>Want to join our team?</h2>
        <p>Do you have ideas that you think may help us do a better science? Then...</p>
        <a href="contact-us.html" class="btn btn-primary">Contact us</a> </div>
    </div>
  </div>
  <!--/.container--> 
</section>
<!--/#get-started-->
<div class="color-border"> </div>
<footer id="footer">
  <div class="container">
    <div class="row">
      <div class="col-sm-6">Copyright &copy; 2015 Goodarzi Lab.
        <br /><small>Design by <a href="http://www.templategarden.com" rel="nofollow">TemplateGarden</a></small>
      </div>
      <div class="col-sm-6">
        <div class="follow-us"> <a class="fa fa-twitter social-icon" href="https://twitter.com/genophoria"></a> <a class="fa fa-linkedin social-icon" href="https://www.linkedin.com/in/hgoodarzi/"></a> </div>
      </div>
    </div>
  </div>
</footer>
<!--/#footer--> 

<script src="js/jquery.js"></script> 
<script src="js/bootstrap.min.js"></script> 
<script src="js/jquery.prettyPhoto.js"></script> 
<script src="js/jquery.isotope.min.js"></script> 
<script src="js/main.js"></script>
</body>
</html>
