<!DOCTYPE html>
<html lang="en">
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-135435972-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-135435972-1');
</script>

<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Goodarzi Lab @ UCSF</title>
<meta name="description" content="Goodarzi Lab @ UCSF">

<!-- core CSS -->
<link href="css/bootstrap.min.css" rel="stylesheet">
<link href="css/bootstrap-social.css" rel="stylesheet">
<link rel="stylesheet" href="css/academicons.min.css"/>
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
<link href="css/prettyPhoto.css" rel="stylesheet">
<link href="css/main.css" rel="stylesheet">
<link href="css/responsive.css" rel="stylesheet">
<link rel="shortcut icon" type="image/x-icon" href="favicon-flask.ico" />
<!--[if lt IE 9]>
    <script src="js/html5shiv.js"></script>
    <script src="js/respond.min.js"></script>
    <![endif]-->
</head>
<!--/head-->
<body>
<header id="header">
  <nav class="navbar navbar-inverse" role="banner">
    <div class="container">
      <div class="navbar-header">
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
        <a class="navbar-brand" href="index.html"><i class="fa fa-flask"></i> Goodarzi Lab</a> 
      </div>
      <div class="collapse navbar-collapse navbar-right">
        <ul class="nav navbar-nav">
          <li><a href="index.html">Home</a></li>
          <li class="active"><a href="research.html">Research</a></li>
          <li><a href="publications.html">Publications</a></li>
          <li><a href="people.html">People</a></li>
          <li><a href="software.html">Software</a></li>
          <li><a href="donate.html">Donate</a></li>
          <li><a href="positions.html">Positions</a></li>
          <li><a href="contact-us.html">Contact</a></li>
        </ul>
      </div>
    </div>
    <!--/.container--> 
  </nav>
  <!--/nav--> 
  
</header>
<!--/header-->
<div class="color-border"> </div>
<section>
  <div class="container">
    <div class="center">
      <h2>Research</h2>
      <p class="lead">We combine modern experimental and computational technologies to understand complex human diseases at a molecular level.<br />
        We are predominantly focused on metastatic progression in multiple cancers and neurodegenerative diseases as the biggest challenges to human health in the 21st century.</p>
    </div>
    <!--/.center-->
    <div class="row box-shadow">
      <div class="col-xs-12 col-sm-12 col-md-12">
        <div class="row thumbnail row-sm-flex-center">
          <div class="col-xs-4 col-sm-4 col-md-3">
            <img class="img-responsive" src="images/img/scRNA-seq.jpg">
          </div>
          <div class="col-xs-8 col-sm-8 col-md-9">
            <h2>Dissecting the evolutionary dynamics of cancer progression and tumor-environment interactions</h2>
            <p>Over the past decade, cancer progression has emerged as a complex evolutionary process with many dynamics forces at play at every step. The resulting widespread reprogramming of the gene expression landscape in cancer cells is a hallmark of cancer development. While the focus of cancer biologists has been on the key signaling pathways and regulatory programs that are hijacked by cancer cells, my group has been interested in the possibility of emergent regulatory modules that are engineered by cancer cells and fall outside of existing regulatory networks. This question led us to the discovery of orphan non-coding RNAs (oncRNAs), a class of small non-coding RNAs that are generally not expressed in normal tissue. We have demonstrated that cancer cells can adopt oncRNAs to carry out new regulatory functions that promote metastatic progression.</p>

            <p>An evolutionary view of cancer progression, also requires capturing the population dynamics and identifying the “attractor” regulatory states that are most metastatic. We are taking advantage of mouse models of metastasis, in conjunction with cutting-edge technologies that are built on single-cell profiling, to reveal the complex and intricate evolutionary trajectories that lead to metastatic dissemination. We have observed that the notion of “tumor heterogeneity” is most likely the result of multiple parallel (and possibly competing) survival strategies (<a href="http://www.nature.com/ncomms/2016/160503/ncomms11246/full/ncomms11246.html">Nguyen et al, 2016, <i>Nature Comm</i></a>). More importantly, these strategies are specific to the distal organ microenvironment as well interactions with stromal and immune cells. 
            </p>
            <ul>
              <li>Fish L, Zhang S, Yu J, Culbertson B, Zhou A, Goga A, <strong>Goodarzi H</strong> (2018). Cancer cells exploit an orphan RNA to drive metastatic progression. <strong><em>Nature Med</em></strong> 24, 1743-51. <a href="https://www.nature.com/articles/s41591-018-0230-4"><i class="fa fa-chrome"></i></a></li>
              <li>Nguyen A, Yoshida M, <strong>Goodarzi H</strong>, Tavazoie SF (2016). Highly variable cancer subpopulations that exhibit enhanced transcriptome variability and metastatic fitness. <strong><em>Nature Comm</em></strong> 7: 11246.<a href="http://www.nature.com/ncomms/2016/160503/ncomms11246/full/ncomms11246.html"><i class="fa fa-chrome"></i></a></li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div class="row box-shadow">
      <div class="col-xs-12 col-sm-12 col-md-12">
        <div class="row thumbnail row-sm-flex-center">
          <div class="col-xs-4 col-sm-4 col-md-3">
            <img class="img-responsive" src="images/img/platform.jpg">
          </div>
          <div class="col-xs-8 col-sm-8 col-md-9">
            <h2>Discovering novel post-transcriptional regulatory programs that drive human disease</h2>
            <p>Complex human pathologies, such as cancer and neurodegenerative diseases, accompany widespread dysregulations in the regulatory programs that govern gene expression dynamics. A major component of my research is focused on unbiased and systematic platforms that enable the discovery of mechanistically novel post-transcriptional regulatory pathways that contribute to disease progression. For example, by focusing on small non-coding RNAs that are induced under stress, we identified a novel of class of tRNA-derived tRNA fragments (tiRNAs) that act as suppressors of breast cancer metastasis (<a href="http://www.cell.com/cell/abstract/S0092-8674(15)00318-9">Goodarzi et al, 2015, <i>Cell</i></a>). Recently, we also reported the discovery a post-transcriptional regulatory pathway that was not only mechanistically novel, but also directly promotes breast cancer metastasis (<a href="Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins">Goodarzi et al, 2014, <i>Nature</i></a>). These discoveries were made possible with development of integrated strategies, which combine modern experimental and computational technologies. This interdisciplinary approach, which taps into my background as a computational and experimental biologist, is crucial for tackling complex phenotypes in human disease.</p>
            <ul>
              <li>Yu J, Naviskas A, Asgharian H, Culbertson B, Fish L, Garcia K, Olegario JP, Dermit M, Dodel M, Hanisch B, Liu Y, Weinberg EM, Dienstmann R, Warren RS, Mardakheh F, and <strong>Goodarzi H</strong> (2020). RBMS1 suppresses colon cancer metastasis through targeted stabilization of its mRNA regulon. <strong><em>Cancer Disc</em></strong> 10.1158/2159-8290.CD-19-1375 <a href="https://www.biorxiv.org/content/10.1101/2020.01.22.916205v1"><i class="ai ai-biorxiv"></i></a>  <a href="https://cancerdiscovery.aacrjournals.org/content/early/2020/06/06/2159-8290.CD-19-1375"><i class="fa fa-chrome"></i></a></li>
              <li>Fish L, Navickas A, Culbertson B, Xu Y, Nguyen HCB, Zhang S, Hochman M, Okimoto R, Dill BD, Molina H, Najafabadi HS, Alarcon C, Ruggero D, and <strong>Goodarzi H</strong> (2018). Nuclear TARBP2 Drives Oncogenic Dysregulation of RNA Splicing and Decay. <strong><em>Mol Cell</em></strong> doi:10.1016/j.molcel.2019.06.001. <a href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(19)30434-4"><i class="fa fa-chrome"></i></a></li>
              <li><strong>Goodarzi H</strong>*&#x271D, Nguyen HCB, Zhang S, Dill BD, Molina H, Tavazoie SF&#x271D (2016). Modulated expression of specific tRNAs drives gene expression and cancer progression. <strong><em>Cell</em></strong> 165, 1416-1427. *Contributed equally; &#x271D Co-corresponding authors. <a href="http://www.cell.com/cell/fulltext/S0092-8674(16)30649-3"><i class="fa fa-chrome"></i></a></li>
              <li><strong>Goodarzi H</strong>, Liu X, Nguyen HCB, Fish L, Tavazoie SF (2015). Endogenous tRNA-Derived Fragments Suppress Breast Cancer Progression via YBX1 Displacement. <strong><em>Cell</em></strong> 161: 790-802. <a href="http://www.cell.com/cell/abstract/S0092-8674(15)00318-9"><i class="fa fa-chrome"></i></a></li>
              <li><strong>Goodarzi H</strong>, Zhang S, Buss CG, Fish L, Tavazoie S, Tavazoie SF (2014). Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins. <strong><em>Nature</em></strong> 513, 255-260 <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature13466.html"><i class="fa fa-chrome"></i></a></li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div class="row box-shadow">
      <div class="col-xs-12 col-sm-12 col-md-12">
        <div class="row thumbnail row-sm-flex-center">
          <div class="col-xs-4 col-sm-4 col-md-3">
            <img class="img-responsive" src="images/img/temporal.jpg">
          </div>
          <div class="col-xs-8 col-sm-8 col-md-9">
            <h2>Post-transcriptional regulators and their roles in neuronal development and neurodegenerative disease</h2>
            <p>Brain differs extensively from other tissues with respect to its transcriptome profile, with large sets of genes specifically expressed or inhibited in the brain. Among the regulators of brain-specific gene expression programs, however, factors that determine the stability and decay of mRNAs are surprisingly understudied. MicroRNAs (miRNAs) and RNA-binding proteins (RBPs) are the largest classes of such factors, which often function through binding to specific sequences within the 3’ untranslated region (UTR) of their target mRNAs – binding of miRNAs often leads to mRNA destabilization, and the binding of RBPs can either stabilize or destabilize the mRNA. There is increasing evidence that these factors play a pivotal role in shaping the transcriptome and identity of brain cells, and that deficits in their function is linked to various neurodevelopmental and neurodegenerative disorders, including Alzheimer’s disease (AD).</p>

            <p>In a collaborative effort involving the Fattahi (UCSF) and Najafabadi (McGill) labs, we are developing novel computational frameworks as well as robust experimental models to study the contribution of RNA-binding proteins and other post-transcriptional regulators to neuronal development and neurodegenrative disease.</p>
            <ul>
              <li>Alkallas R, Fish L, <strong>Goodarzi H</strong>, Najafabadi HS (2017). Inference of RNA decay rate from transcriptional profiling highlights the regulatory programs of Alzheimer’s disease. <strong><em>Nature Comm</em></strong>  8: 909. <a href="https://www.nature.com/articles/s41467-017-00867-z"><i class="fa fa-chrome"></i></a></li>
              <li>Hwang HW, Park CY, <strong>Goodarzi H</strong>, Fak JJ, Mele A, Moore MJ, Saito Y, Darnell RB (2016). PAPERCLIP Identifies MicroRNA Targets and a Role of CstF64/64tau in Promoting Non-canonical poly(A) Site Usage. <strong><em>Cell Rep</em></strong> 15(2):423-435.<a href="http://www.sciencedirect.com/science/article/pii/S2211124716302649"><i class="fa fa-chrome"></i></a></li>
              <li>Chiu IM, Morimoto ETA, <strong>Goodarzi H</strong>, et al., Maniatis T (2013). A Neurodegeneration-Specific Gene-Expression Signature of Acutely Isolated Microglia from an Amyotrophic Lateral Sclerosis Mouse Model. <strong><em>Cell Rep</em></strong> 4(2): 385-401 <a href="http://www.cell.com/cell-reports/fulltext/S2211-1247(13)00296-9"><i class="fa fa-chrome"></i></a></li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div class="row box-shadow">
      <div class="col-xs-12 col-sm-12 col-md-12">
        <div class="row thumbnail row-sm-flex-center">
          <div class="col-xs-4 col-sm-4 col-md-3">
            <img class="img-responsive" src="images/img/ssRNA.jpg">
          </div>
          <div class="col-xs-8 col-sm-8 col-md-9">
            <h2>Computational discovery and characterization of structural regulatory elements in RNA</h2>
            <p>A systematic approach to cis-regulatory element discovery in RNA requires capturing the information provided by both the structure and the underlying sequence. The inability of motif discovery methodologies to seamlessly incorporate structural information as part of their search algorithms significantly hinders the identification of structural elements in RNA. To address the challenge outlined above, we have implemented, and continue to expand, a computational framework for discovering structural RNA elements that govern the behavior of RNA in the cell. In this approach, named TEISER (tool for eliciting informative structural elements in RNA), the large space of small structural seeds is systematically explored to identify elements that are significantly informative of transcriptomic measurements (<a href="http://www.nature.com/nature/journal/v485/n7397/full/nature11013.html">Goodarzi et al, 2012, <i>Nature</i></a>). Using this approach, we have identified a number of structural elements that play a direct role in gene expression regulation and  disease (e.g. <a href="Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins">Goodarzi et al, 2014, <i>Nature</i></a>).</p>
            <ul>
              <li><strong>Goodarzi H</strong>, Najafabadi HS, Oikonomou P, Greco TM, Fish L, Salavati R, Cristea IM, Tavazoie S (2012).  Systematic discovery of structural elements governing stability of mammalian messenger RNAs. <strong><em>Nature</em></strong> 485, 264-268. <a href="http://www.nature.com/nature/journal/v485/n7397/full/nature11013.html"><i class="fa fa-chrome"></i></a></li>
              <li><strong>Goodarzi H</strong>, Zhang S, Buss CG, Fish L, Tavazoie S, Tavazoie SF (2014). Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins. <strong><em>Nature</em></strong> 513, 255-260 <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature13466.html"><i class="fa fa-chrome"></i></a></li>
              <li>Oikonomou P*, <strong>Goodarzi H</strong>*, Tavazoie S (2014). Systematic Identification of Regulatory Elements in Conserved 3? UTRs of Human Transcripts. <strong><em>Cell Rep</em></strong> 7(1): 281-292. *Contributed equally <a href="http://www.cell.com/cell-reports/fulltext/S2211-1247(14)00159-4"><i class="fa fa-chrome"></i></a></li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <!--/.row--> 
    <div class="row box-shadow">
      <div class="col-xs-12 col-sm-12 col-md-12">
        <div class="row thumbnail row-sm-flex-center">
          <div class="col-xs-4 col-sm-4 col-md-3">
            <img class="img-responsive" src="images/img/c3ulib.jpg">
          </div>
          <div class="col-xs-8 col-sm-8 col-md-9">
            <h2>Developing integrated computational and experimental technologies to dissect regulatory interactions </h2>
            <p>Deciphering the noncoding regulatory genome is a formidable challenge. Despite the wealth of available gene expression data, broadly applicable methods for characterizing the regulatory elements that shape the underlying dynamics have been in short supply. To overcome this challenge, we have developed a suite of integrated computational and experimental techniques that overcome the major obstacles in revealing the regulatory logic underlying RNA dynamics in the cell under normal and pathologic conditions. Our computational frameworks for detecting linear and structural regulatory DNA and RNA motifs rely on directly assessing the mutual information between sequence and whole-transcriptomic measurements. Our approach makes minimal assumptions about the background sequence model and the mechanisms by which elements affect gene expression. In parallel, we have developed a series of experimental strategies, based on whole-genome observations, to validate and functionally probe these regulatory interactions in vivo. While our findings provide an encyclopedic snapshot of regulatory interactions in the cell, our knowledge of the regulatory genome is still in its infancy. Applying these strategies to other experimental models is a crucial step towards a more comprehensive understanding of the regulatory genome.</p>
            <ul>
              <li>Oikonomou P*, <strong>Goodarzi H</strong>*, Tavazoie S (2014). Systematic Identification of Regulatory Elements in Conserved 3? UTRs of Human Transcripts. <strong><em>Cell Reports</em></strong> 7(1): 281-292. *Contributed equally <a href="http://www.cell.com/cell-reports/fulltext/S2211-1247(14)00159-4"><i class="fa fa-chrome"></i></a></li>
              <li><strong>Goodarzi H</strong>*, Elemento O*, Tavazoie S (2009). Revealing Global Regulatory Perturbations across Human Cancers. <strong><em>Mol Cell</em></strong> 36:900-11. *contributed equally <a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(09)00857-0"><i class="fa fa-chrome"></i></a></li>
              <li>Najafabadi HS*, <strong>Goodarzi H</strong>*, Salavati R (2009).  Universal function-specificity of codon usage. <strong><em>Nucl Acids Res</em></strong> 37(21):7014-7023. *contributed equally <a href="http://nar.oxfordjournals.org/cgi/content/full/gkp792"><i class="fa fa-chrome"></i></a></li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div class="row box-shadow">
      <div class="col-xs-12 col-sm-12 col-md-12">
        <div class="row thumbnail row-sm-flex-center">
          <div class="col-xs-4 col-sm-4 col-md-3">
            <img class="img-responsive" src="images/img/m6A.jpg">
          </div>
          <div class="col-xs-8 col-sm-8 col-md-9">
            <h2>Dissecting the role of epitranscriptomic modifications in RNA dynamics</h2>
            <p>N<sup>6</sup>-methyladenosine (m6A) has been recently identified as an epitranscriptomic modification of mRNAs in eukaryotes, but its regulatory consequences and functional role in the cell is largely uncharacterized. In a series of studies, we have depicted a pivotal role for m6A modifications in miRNA processing. Using computational tools and focused experimental techniques, we have demonstrated that this modification marks the sites of primary miRNAs and helps recruit the miRNA machinery. We successfully identified the RNA-binding protein HNRNPA2B1 as one nuclear reader of this modification, which initiates the processing by interacting with DGCR8. In our view, this but one example of RNA editing regulating key RNA processing events in the cell. As such, we are interested in understanding how RNA methylation is initiated, what is its impact on the targets RNA molecule, and how this effects is brought about.</p>
            <ul>
              <li>Alarcon C, <strong>Goodarzi H</strong>, Lee H, Liu X, Tavazoie S, Tavazoie SF (2015). HNRNPA2B1 Is a Mediator of m6A-Dependent Nuclear RNA Processing Events. <strong><em>Cell</em></strong> 162: 1299-1308. <a href="http://www.cell.com/cell/abstract/S0092-8674(15)01024-7"><i class="fa fa-chrome"></i></a></li>
              <li>Alarcon CR, Lee H*, <strong>Goodarzi H</strong>*, Halberg N, Tavazoie S (2015). N<sup>6</sup>-methyladenosine marks primary microRNAs for processing. <strong><em>Nature</em></strong> 519, 482-485. *Contributed equally <a href="http://www.nature.com/nature/journal/v519/n7544/full/nature14281.html?WT.ec_id=NATURE-20150326"><i class="fa fa-chrome"></i></a></li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
  <!--/.container--> 
<!--/#research--> 
</section>
    
<section id="get-started">
  <div class="container">
    <div class="row">
      <div class="col-sm-12">
        <h2>Want to join our team?</h2>
        <p>Do you have ideas that you think may help us do a better science? Then...</p>
        <a href="contact-us.html" class="btn btn-primary">Contact us</a> </div>
    </div>
  </div>
  <!--/.container--> 
</section>
<!--/#get-started-->
<div class="color-border"> </div>
<footer id="footer">
  <div class="container">
    <div class="row">
      <div class="col-sm-6">Copyright &copy; 2015 Goodarzi Lab.
        <br /><small>Design by <a href="http://www.templategarden.com" rel="nofollow">TemplateGarden</a></small>
      </div>
      <div class="col-sm-6">
        <div class="follow-us"> <a class="fa fa-twitter social-icon" href="https://twitter.com/genophoria"></a> <a class="fa fa-linkedin social-icon" href="https://www.linkedin.com/in/hgoodarzi/"></a> </div>
      </div>
    </div>
  </div>
</footer>
<!--/#footer--> 

<script src="js/jquery.js"></script> 
<script src="js/bootstrap.min.js"></script> 
<script src="js/jquery.prettyPhoto.js"></script> 
<script src="js/jquery.isotope.min.js"></script> 
<script src="js/main.js"></script>
</body>
</html>
