<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Goodarzi Lab @ UCSF</title>
<meta name="description" content="Goodarzi Lab @ UCSF">

<!-- core CSS -->
<link href="css/bootstrap.min.css" rel="stylesheet">
<link href="css/bootstrap-social.css" rel="stylesheet">
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
<link href="css/prettyPhoto.css" rel="stylesheet">
<link href="css/main.css" rel="stylesheet">
<link href="css/responsive.css" rel="stylesheet">
<link rel="shortcut icon" type="image/x-icon" href="favicon-flask.ico" />
<!--[if lt IE 9]>
    <script src="js/html5shiv.js"></script>
    <script src="js/respond.min.js"></script>
    <![endif]-->
</head>
<!--/head-->
<body>
<header id="header">
  <nav class="navbar navbar-inverse" role="banner">
    <div class="container">
      <div class="navbar-header">
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
        <a class="navbar-brand" href="index.html"><i class="fa fa-flask"></i> Goodarzi Lab</a> 
      </div>
      <div class="collapse navbar-collapse navbar-right">
        <ul class="nav navbar-nav">
          <li><a href="index.html">Home</a></li>
          <li class="active"><a href="research.html">Research</a></li>
          <li><a href="publications.html">Publications</a></li>
          <li><a href="people.html">People</a></li>
          <li><a href="software.html">Software</a></li>
          <li><a href="https://genophoria.wordpress.com/">Blog</a></li>
          <li><a href="contact-us.html">Contact</a></li>
        </ul>
      </div>
    </div>
    <!--/.container--> 
  </nav>
  <!--/nav--> 
  
</header>
<!--/header-->
<div class="color-border"> </div>
<section>
  <div class="container">
    <div class="center">
      <h2>Research</h2>
      <p class="lead">We combine modern experimental and computational technologies to understand cancer progression at a molecular level.<br />
        Our goal is to develop novel strategies for studying, diagnosing, and ultimately treating cancer.</p>
    </div>
    <!--/.center-->
    <div class="row box-shadow">
      <div class="col-xs-12 col-sm-12 col-md-12">
        <div class="row thumbnail row-sm-flex-center">
          <div class="col-xs-4 col-sm-4 col-md-3">
            <img class="img-responsive" src="images/img/TARBP2.jpg">
          </div>
          <div class="col-xs-8 col-sm-8 col-md-9">
            <h2>Discovering novel post-transcriptional regulatory programs that drive cancer metastasis</h2>
            <p>For cancer to spread, genes involved in the metastatic process must be altered. This deregulation largely arises from modulations in the regulatory programs that govern gene expression dynamics. A major component of my research is focused on unbiased and systematic platforms that enable the discovery of mechanistically novel post-transcriptional regulatory pathways that contribute to metastatic disease. For example, by focusing on small non-coding RNAs that are induced under stress, we identified a novel of class of tRNA-derived tRNA fragments (tiRNAs) that act as suppressors of breast cancer metastasis (<a href="http://www.cell.com/cell/abstract/S0092-8674(15)00318-9">Goodarzi et al, 2015, <i>Cell</i></a>). Recently, we also reported the discovery a post-transcriptional regulatory pathway that was not only mechanistically novel, but also directly promotes breast cancer metastasis (<a href="Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins">Goodarzi et al, 2014, <i>Nature</i></a>). These discoveries were made possible with development of integrated strategies, which combine modern experimental and computational technologies. This interdisciplinary approach, which taps into my background as a computational and experimental biologist, is crucial for tackling complex phenotypes in human disease.</p>
            <ul>
              <li><strong>Goodarzi H</strong>*&#x271D, Nguyen HCB, Zhang S, Dill BD, Molina H, Tavazoie SF&#x271D (2016). Modulated expression of specific tRNAs drives gene expression and cancer progression. <strong><em>Cell</em></strong> 165, 1416-1427. *Contributed equally; &#x271D Co-correspoding authors. <a href="http://www.cell.com/cell/fulltext/S0092-8674(16)30649-3"><i class="fa fa-chrome"></i></a></li>
              <li><strong>Goodarzi H</strong>, Liu X, Nguyen HCB, Fish L, Tavazoie SF (2015). Endogenous tRNA-Derived Fragments Suppress Breast Cancer Progression via YBX1 Displacement. <strong><em>Cell</em></strong> 161: 790-802. <a href="http://www.cell.com/cell/abstract/S0092-8674(15)00318-9"><i class="fa fa-chrome"></i></a></li>
              <li><strong>Goodarzi H</strong>, Zhang S, Buss CG, Fish L, Tavazoie S, Tavazoie SF (2014). Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins. <strong><em>Nature</em></strong> 513, 255-260 <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature13466.html"><i class="fa fa-chrome"></i></a></li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div class="row box-shadow">
      <div class="col-xs-12 col-sm-12 col-md-12">
        <div class="row thumbnail row-sm-flex-center">
          <div class="col-xs-4 col-sm-4 col-md-3">
            <img class="img-responsive" src="images/img/ssRNA.jpg">
          </div>
          <div class="col-xs-8 col-sm-8 col-md-9">
            <h2>Computational discovery and characterization of structural regulatory elements in RNA</h2>
            <p>A long-standing obstacle in decoding post-transcriptional programs is accounting for the contribution of local secondary structures in RNA-mediated regulatory interactions. A systematic approach to cis-regulatory element discovery in RNA requires capturing the information provided by both the structure and the underlying sequence. The inability of motif discovery methodologies to seamlessly incorporate structural information as part of their search algorithms significantly hinders the identification of structural elements in RNA. To address the challenge outlined above, we have implemented, and continue to expand, a computational framework for discovering structural RNA elements that govern the behavior of RNA in the cell. In this approach, named TEISER (tool for eliciting informative structural elements in RNA), the large space of small structural seeds is systematically explored to identify elements that are significantly informative of transcriptomic measurements (<a href="http://www.nature.com/nature/journal/v485/n7397/full/nature11013.html">Goodarzi et al, 2012, <i>Nature</i></a>). Using this approach, we have identified a number of structural elements that play a direct role in gene expression regulation and human disease (e.g. <a href="Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins">Goodarzi et al, 2014, <i>Nature</i></a>). Going forward, we continue to expand on the TEISER framework by integrating both computational and experimental strategies to boost its statistical power.</p>
            <ul>
              <li><strong>Goodarzi H</strong>, Najafabadi HS, Oikonomou P, Greco TM, Fish L, Salavati R, Cristea IM, Tavazoie S (2012).  Systematic discovery of structural elements governing stability of mammalian messenger RNAs. <strong><em>Nature</em></strong> 485, 264-268. *Contributed equally; &#x271D Co-correspoding authors. <a href="http://www.nature.com/nature/journal/v485/n7397/full/nature11013.html"><i class="fa fa-chrome"></i></a></li>
              <li><strong>Goodarzi H</strong>, Zhang S, Buss CG, Fish L, Tavazoie S, Tavazoie SF (2014). Metastasis-suppressor transcript destabilization through TARBP2 binding of mRNA hairpins. <strong><em>Nature</em></strong> 513, 255-260 <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature13466.html"><i class="fa fa-chrome"></i></a></li>
              <li>Oikonomou P*, <strong>Goodarzi H</strong>*, Tavazoie S (2014). Systematic Identification of Regulatory Elements in Conserved 3? UTRs of Human Transcripts. <strong><em>Cell Reports</em></strong> 7(1): 281-292. *Contributed equally <a href="http://www.cell.com/cell-reports/fulltext/S2211-1247(14)00159-4"><i class="fa fa-chrome"></i></a></li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <!--/.row--> 
    <div class="row box-shadow">
      <div class="col-xs-12 col-sm-12 col-md-12">
        <div class="row thumbnail row-sm-flex-center">
          <div class="col-xs-4 col-sm-4 col-md-3">
            <img class="img-responsive" src="images/img/c3ulib.jpg">
          </div>
          <div class="col-xs-8 col-sm-8 col-md-9">
            <h2>Developing integrated computational and experimental technologies to dissect regulatory interactions </h2>
            <p>Deciphering the noncoding regulatory genome is a formidable challenge. Despite the wealth of available gene expression data, broadly applicable methods for characterizing the regulatory elements that shape the underlying dynamics have been in short supply. To overcome this challenge, we have developed a suite of integrated computational and experimental techniques that overcome the major obstacles in revealing the regulatory logic underlying RNA dynamics in the cell under normal and pathologic conditions. Our computational frameworks for detecting linear and structural regulatory DNA and RNA motifs rely on directly assessing the mutual information between sequence and whole-transcriptomic measurements. Our approach makes minimal assumptions about the background sequence model and the mechanisms by which elements affect gene expression. In parallel, we have developed a series of experimental strategies, based on whole-genome observations, to validate and functionally probe these regulatory interactions in vivo. While our findings provide an encyclopedic snapshot of regulatory interactions in the cell, our knowledge of the regulatory genome is still in its infancy. Applying these strategies to other experimental models is a crucial step towards a more comprehensive understanding of the regulatory genome.</p>
            <ul>
              <li>Oikonomou P*, <strong>Goodarzi H</strong>*, Tavazoie S (2014). Systematic Identification of Regulatory Elements in Conserved 3? UTRs of Human Transcripts. <strong><em>Cell Reports</em></strong> 7(1): 281-292. *Contributed equally <a href="http://www.cell.com/cell-reports/fulltext/S2211-1247(14)00159-4"><i class="fa fa-chrome"></i></a></li>
              <li><strong>Goodarzi H</strong>*, Elemento O*, Tavazoie S (2009). Revealing Global Regulatory Perturbations across Human Cancers. <strong><em>Mol Cell</em></strong> 36:900-11. *contributed equally <a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(09)00857-0"><i class="fa fa-chrome"></i></a></li>
              <li>Najafabadi HS*, <strong>Goodarzi H</strong>*, Salavati R (2009).  Universal function-specificity of codon usage. <strong><em>Nucl Acids Res</em></strong> 37(21):7014-7023. *contributed equally <a href="http://nar.oxfordjournals.org/cgi/content/full/gkp792"><i class="fa fa-chrome"></i></a></li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div class="row box-shadow">
      <div class="col-xs-12 col-sm-12 col-md-12">
        <div class="row thumbnail row-sm-flex-center">
          <div class="col-xs-4 col-sm-4 col-md-3">
            <img class="img-responsive" src="images/img/m6A.jpg">
          </div>
          <div class="col-xs-8 col-sm-8 col-md-9">
            <h2>Dissecting the role of epitranscriptomic modifications in RNA dynamics</h2>
            <p>N<sup>6</sup>-methyladenosine (m6A) has been recently identified as an epitranscriptomic modification of mRNAs in eukaryotes, but its regulatory consequences and functional role in the cell is largely uncharacterized. In a series of studies, we have depicted a pivotal role for m6A modifications in miRNA processing. Using computational tools and focused experimental techniques, we have demonstrated that this modification marks the sites of primary miRNAs and helps recruit the miRNA machinery. We successfully identified the RNA-binding protein HNRNPA2B1 as one nuclear reader of this modification, which initiates the processing by interacting with DGCR8. In our view, this but one example of RNA editing regulating key RNA processing events in the cell. As such, we are interested in understanding how RNA methylation is initiated, what is its impact on the targets RNA molecule, and how this effects is brought about.</p>
            <ul>
              <li>Alarcon C, <strong>Goodarzi H</strong>, Lee H, Liu X, Tavazoie S, Tavazoie SF (2015). HNRNPA2B1 Is a Mediator of m6A-Dependent Nuclear RNA Processing Events. <strong><em>Cell</em></strong> 162: 1299-1308. <a href="http://www.cell.com/cell/abstract/S0092-8674(15)01024-7"><i class="fa fa-chrome"></i></a></li>
              <li>Alarcon CR, Lee H*, <strong>Goodarzi H</strong>*, Halberg N, Tavazoie S (2015). N<sup>6</sup>-methyladenosine marks primary microRNAs for processing. <strong><em>Nature</em></strong> 519, 482-485. *Contributed equally <a href="http://www.nature.com/nature/journal/v519/n7544/full/nature14281.html?WT.ec_id=NATURE-20150326"><i class="fa fa-chrome"></i></a></li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
  <!--/.container--> 
<!--/#research--> 
</section>
    
<section id="get-started">
  <div class="container">
    <div class="row">
      <div class="col-sm-12">
        <h2>Want to join our team?</h2>
        <p>Do you have ideas that you think may help us do a better science? Then...</p>
        <a href="contact-us.html" class="btn btn-primary">Contact us</a> </div>
    </div>
  </div>
  <!--/.container--> 
</section>
<!--/#get-started-->
<div class="color-border"> </div>
<footer id="footer">
  <div class="container">
    <div class="row">
      <div class="col-sm-6">Copyright &copy; 2015 Goodarzi Lab.
        <br /><small>Design by <a href="http://www.templategarden.com" rel="nofollow">TemplateGarden</a></small>
      </div>
      <div class="col-sm-6">
        <div class="follow-us"> <a class="fa fa-facebook social-icon" href="#"></a> <a class="fa fa-twitter social-icon" href="#"></a> <a class="fa fa-linkedin social-icon" href="#"></a> <a class="fa fa-google-plus social-icon" href="#"></a> </div>
      </div>
    </div>
  </div>
</footer>
<!--/#footer--> 

<script src="js/jquery.js"></script> 
<script src="js/bootstrap.min.js"></script> 
<script src="js/jquery.prettyPhoto.js"></script> 
<script src="js/jquery.isotope.min.js"></script> 
<script src="js/main.js"></script>
</body>
</html>
